populus/contracts/functions.py

```diff
@@ -59,7 +59,16 @@ def sendTransaction(self, *args, **kwargs):
         data = self.get_call_data(args)
 
         if 'gas' not in kwargs:
-            kwargs['gas'] = self.contract._meta.blockchain_client.get_max_gas()
+            # The gasLimit value on the geth chain seems to continuously
+            # decline after every block. We use a value of gas 
+            # slightly less than the get_max_gas value so that when 
+            # the transaction gets processed, the gas value we send 
+            # is still less than the gasLimit value when our transaction 
+            # eventually gets processed. 
+            gasLimitFraction = 0.9
+            kwargs['gas'] = int(
+                gasLimitFraction * self.contract._meta.blockchain_client.get_max_gas()
```

Comment:
I know this function is named with this naming convention but can you change the variable to be `GAS_LIMIT_FRACTION` and declare it at the top level of the module.